packages/@glimmer/application/src/loaders/bytecode/resolver.ts

```diff
@@ -56,6 +56,7 @@ export default class BytecodeResolver implements RuntimeResolver<TemplateMeta> {
   lookupComponentDefinition(name: string, referrer: TemplateMeta): ComponentDefinition {
     let owner = this.owner;
     let manager = this.managerFor();
+    referrer = referrer || { specifier: null };
```

Comment:
Will this produce a confusing error if you call `renderToString` with an unknown component name? E.g. "invoked using the {{component}} helper in null".

Reply:
We may want to enforce as an invariant that `referrer` can never be null. Are there other cases I'm forgetting?

Reply:
Ahh, apologies. Forgot about that case.